packages/grid/x-data-grid-pro/src/material/index.ts

```diff
@@ -0,0 +1,15 @@
+import type { GridProIconSlotsComponent } from '../models';
+import { GridPushPinRightIcon, GridPushPinLeftIcon } from './icons';
+import { GridProColumnMenu } from '../components/GridProColumnMenu';
+
+const iconSlots: GridProIconSlotsComponent = {
+  ColumnMenuPinRightIcon: GridPushPinRightIcon,
+  ColumnMenuPinLeftIcon: GridPushPinLeftIcon,
+};
+
+const materialSlots = {
+  ...iconSlots,
+  ColumnMenu: GridProColumnMenu,
```

Comment:
Should `ColumnMenu` be included in `material` slots for pro and premium packages keeping in mind it's a custom grid component?
(It isn't added for MIT package though)

Reply:
Right, I forgot to include it in the MIT package.
Or is your concern that `ColumnMenu` slot shouldn't be treated as material component?

Reply:
Yes, that was my thinking, like other components (Cell, Row, etc), ColumnMenu is not essentially a material component and might be treated as a Grid component.

Would MenuItem and other material slots being used in ColumnMenu be sufficient to support Joy UI? Or you think that we should also consider ColumnMenu to be part of material slots.

Reply:
Right, `MenuList` and `MenuItem` should be enough for this, and Jun already started the extraction of those slots in https://github.com/mui/mui-x/pull/8059/files#r1119307165 👍